setup.py

```diff
@@ -14,5 +14,5 @@
         "Programming Language :: Python",
     ],
     packages=setuptools.find_packages() + ["contracts", "migrations"],
-    install_requires=["py-solc-x", "web3", "hmt-basemodels", "boto3"],
+    install_requires=["py-solc-x", "trinity", "hmt-basemodels", "boto3", "sphinx"],
```

Comment:
I think we also need the specific web3 version here btw.

Also you will need to bump the hmt-escrow version in this file

Reply:
So, web3 is contained in trinity now, but yes, hmt-escrow version is must. Will add that now.

Reply:
Ah I see, but not specifying a trinity version will always get you the latest in downstream packages, don't think we want that.

Reply:
Actually, we kinda want it, since trinity has been fixed for some focal python bug only a few days ago, and the latest release does not contain that patch, yet.

Reply:
The problem it introduces is that it will allow future installs to get free major upgrades, which may have different API's and thus break code that depends on this. Specifying at least a major version is important.

Reply:
Will it do if I solemnly swear to specify a major version as soon as the commit from yesterday in master of trinity will get released?

Reply:
Yeah definitely, I would expect their release process to have labeled "latest" and the version at the same time but maybe that didnt happen.